src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_mon calendar--month-length_31">
```

Comment:
We are using the Two dashes naming style in BEM so it would be better to name your modifiers like that (2nd item on checklist)
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```